### ISSUE TITLE
We need to solve the issue of "clone" packages... This is getting silly.

### ISSUE THREAD

**Issue by Siyfion** [percolatestudio/atmosphere#380]
For example, please let me refer you to this thread: https://github.com/jshimko/meteor-snapsvg/issues/1

Now I'm not proud of the way I've ended up dealing with this, but it's getting stupid. Considering I started off by stating that perhaps we should have collaborated together and both helped maintain the package, it got ugly, fast.

I can see this sort of behaviour becoming more and more of an issue, and people like @jshimko simply won't stop adding to the "clutter" of duplicates in Atmosphere until there's a some kind of enforcement.

**Comment by tmeasday** [comment]
Hi @Siyfion,

Although it's not ideal to have a bunch of duplicates of the same package in the database, I'm not sure there's a whole lot that can be done about it. Just like there are a lot of forks of github repositories, it's just the way things work in an open source world. 

Sure, it would be better if you and @jshimko worked together on a single package, especially for something as simple as a wrapper, but it's not a huge disaster if there are two. Over time, if one is maintained and used by more people, it'll quickly become obvious through download count and stars to end users. Then they can make their own decision.

In short: I wouldn't worry about it.

**Comment by dandv** [comment]
@Siyfion, with Meteor 1.3 supporting direct npm package import, wrapper packages should become less of an issue.

**Comment by Siyfion** [comment]
Agreed @dandv, fantastic news for everyone. I for one am really looking forward to the 1.3 release, going to be a huuuuuge step forward!
